Gemfile-rails-dependencies

```diff
@@ -33,5 +33,10 @@ else
   if version >= '5-1-stable' && RUBY_VERSION  >= "2.3"
     gem "puma"
   end
-  gem 'activerecord-jdbcsqlite3-adapter', platforms: [:jruby]
+
+  if /(\d+)/.match(ENV['RAILS_VERSION'])[1].to_i >= 6
```

Comment:
Not super fan about this. Maybe you have better idea @JonRowe

Reply:
We usually do `ENV['RAILS_VERSION'].to_f >= 6.0`  etc but you do sometimes have to stripe the other chars, I'd probably use `.gsub(/[^\d\.]/,'').to_f` or similar

Reply:
Thanks! Applied in https://github.com/rspec/rspec-rails/pull/2160/commits/fe90da4ea804ea9a52b0402f601cf9479b83ff99

Happy to merge the PR if you are ok.